src/thunderbots/software/ai/world/team.h

```diff
@@ -44,6 +85,15 @@ class Team
      */
     std::size_t size() const;
```

Comment:
`size` seems a bit ambigious, could we maybe call this `numRobots`?

Reply:
Was mainly following conventions for getting the "size" of a container, but `numRobots` seems reasonable